bigtable/client/internal/table_admin.cc

```diff
@@ -141,6 +141,21 @@ void TableAdmin::DropAllRows(std::string table_id, grpc::Status& status) {
 std::string TableAdmin::InstanceName() const {
   return "projects/" + client_->project() + "/instances/" + instance_id_;
 }
+
+::google::bigtable::admin::v2::Snapshot TableAdmin::GetSnapshot(
+    std::string snapshot_name, std::string cluster, grpc::Status& status) {
```

Comment:
Using two parameters of the same type is error prone, somebody (most likely me!) can call this function with the parameters reversed.  Using strong types would be an option:

https://www.fluentcpp.com/2016/12/08/strong-types-for-strong-interfaces/

In any case, please use consistent names for the parameters, these should be `snapshot_id` and `cluster_id`, right?
Also, you are using those parameters as const references only, but taking a value, please change to const reference.